mappings/net/minecraft/class_2906.mapping

```diff
@@ -0,0 +1 @@
+CLASS net/minecraft/class_2906
```

Comment:
Looks wrong. This is just `Identifier` I recall

Reply:
Hmm, that's OverworldChunkGeneratorConfig. Git told me that got deleted >_>

Reply:
Yeah it did. There's no more class_2906. Why did that not get deleted with Drop Invalid Mappings though